code/modules/clothing/chameleon/_chameleon_action.dm

```diff
@@ -120,6 +120,7 @@
 	if(ismob(chameleon_item.loc))
 		var/mob/wearer = chameleon_item.loc
 		wearer.update_clothing(chameleon_item.slot_flags | ITEM_SLOT_HANDS)
+		wearer.refresh_obscured()
```

Comment:
A few places now call `update_clothing` -> `refresh_obscured` which itself conditionally calls `update_clothing` (based on newly obscured slots). I figured it wasn't common enough to be a problem.